Dockerfile

```diff
@@ -1,9 +1,15 @@
 ## This dockerfile builds the client entirely in a Docker context
 
 FROM node:16-alpine AS build
+# Set labels
+LABEL maintainer="Jellyfin Packaging Team - packaging@jellyfin.org"
+LABEL org.opencontainers.image.source="https://github.com/jellyfin/jellyfin-vue"
```

Comment:
Idk much the utility of those, but shouldn't they go the last stage?

Reply:
I've been always seeing those at the top of the Dockerfile, so I assume it's common practice to do so. Do you have any source with common practices for this? [Nothing is specified in the best practices](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#label)

Reply:
I got no problem with, I'm just wondering if it should be present in the last stage as that's the one pushed

Reply:
That doesn't matter, Docker stacks the layers one on top of another, so the last one contains all that are above it.